src/redecomp/utils/MPIUtility.hpp

```diff
@@ -310,7 +310,7 @@ void MPIUtility::Send( const T& container, int dest, int tag ) const
 template <typename T, axom::MemorySpace Sp>
 void MPIUtility::Send( const axom::Array<T, 2, Sp>& container, int dest, int tag ) const
 {
-  MPI_Send( container.shape().m_data, 2, GetMPIType<int>(), dest, tag, comm_ );
+  MPI_Send( container.shape().m_data, 2, GetMPIType<axom::IndexType>(), dest, tag, comm_ );
```

Comment:
Can you clarify the use of axom::IndexType here? We once had our own IndexType. Does axom's handle the 64 bit integer issue under the hood because we are building with that variant? This is compatible with `long` and `long long` used in the MPI Utility functions?